flixel/system/FlxAssets.hx

```diff
@@ -31,7 +32,7 @@ class GraphicVirtualInput extends BitmapData {}
 @:file("assets/images/ui/virtual-input.txt")
 class VirtualInputData extends #if (lime_legacy || nme) ByteArray #else ByteArrayData #end {}
 
-typedef FlxAngelCodeXmlAsset = FlxXmlAsset;
+typedef FlxAngelCodeAsset = OneOfThree<Xml, String, Bytes>;
```

Comment:
how do you use bytes, here? should we instead change [FlxXmlAsset](https://github.com/HaxeFlixel/flixel/blob/6a723096b130f76109c7f0411abfbf63ca634a49/flixel/system/FlxAssets.hx#L49) to accept Bytes? `FlxXmlAsset` has more functionality than `OneOfThree` so this seems like a bit of a downgrade

Reply:
When trying to parse the binary type font descriptor from BMFont, it uses Bytes. The way the bytes are laid out in the file is described here: https://www.angelcode.com/products/bmfont/doc/file_format.html#bin